api/src/test/java/org/openmrs/api/ConceptServiceTest.java

```diff
@@ -2245,9 +2245,27 @@ public void getConceptsByAnswer_shouldFindAnswersForConcept() {
 	 * @see ConceptService#getConceptsByClass(ConceptClass)
 	 */
 	@Test
-	public void getConceptsByClass_shouldNotFailDueToNoNameInSearch() {
-		ConceptService cs = Context.getConceptService();
-		cs.getConceptsByClass(cs.getConceptClass(1));
+	public void getConceptsByClass_shouldGetConceptsByClass() {				
+		// replay
+		List<Concept> actualConcepts = conceptService.getConceptsByClass(new ConceptClass(3));
+		
+		// verify
+		Assert.assertThat(actualConcepts, containsInAnyOrder(conceptService.getConcept(3), conceptService.getConcept(60), conceptService.getConcept(88), conceptService.getConcept(792)));
```

Comment:
You need to assert that the size is exactly 4, I did specify it in my last review:
```java
Assert.assertThat(actualConcepts.size(), is(4));
```

Reply:
How did I miss this! Oohh I'm sorry.

Let me update it!